src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -12572,6 +12558,25 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
 
     // Calculate the IU Te/Tc for the next time step
     this->CalcVRFIUTeTc_FluidTCtrl(state);
+    // update coil and IU evaporating temperature, also keep coil RTF updated with the condenser side cycling ratio, for the FluidTCtrl model
+    for (int VRFTUNum = 1; VRFTUNum <= state.dataHVACVarRefFlow->NumVRFTU; ++VRFTUNum) {
+        auto const &thisTU = state.dataHVACVarRefFlow->VRFTU(VRFTUNum);
+        auto &coolingCoil = state.dataDXCoils->DXCoil(thisTU.CoolCoilIndex);
+        if (this->adjustedTe && (!FirstHVACIteration)) {
+            coolingCoil.EvaporatingTemp = this->EvaporatingTemp;
+            this->IUEvaporatingTemp = this->EvaporatingTemp;
+        }
+
+        int PLF;
+        if (coolingCoil.PLFFPLR(1) > 0 && this->VRFCondCyclingRatio < 1.0) {
+            PLF = Curve::CurveValue(state, coolingCoil.PLFFPLR(1), this->VRFCondCyclingRatio); // Calculate part-load factor
+        } else {
+            PLF = 1.0;
+        }
+        if (coolingCoil.TotalCoolingEnergyRate > 0.0) {
+            coolingCoil.CoolingCoilRuntimeFraction = this->VRFCondCyclingRatio / PLF;
+        }
```

Comment:
I don't see CoolingCoilRuntimeFraction getting set to 0 anywhere. So if TotalCoolingEnergyRate = 0, then CoolingCoilRuntimeFraction will be used from the last time step the compressor operated. I would set CoolingCoilRuntimeFraction = 0.0 in Init.

Reply:
Oh, I see. This is updating the coil model RTF, from the parent object. We should not do that. So why did you have to update the coil RTF? VRFCondCyclingRatio is already passed to SimDXCoil so the coil RTF should already be accurate, if that call is passing the updated cycling ratio.

Reply:
The condenser side cycling ratio is updated in `CalcVRFCondenser_FluidTCtrl`. This took place after SimDXCoil, so the coil RTF is updated here to keep up with the updated condenser side cycling ratio